File: exercicio_8.py
num = int(input("Digite um número para checar: "))

if num > 1:

    for i in range(2, num):

        if (num % i) == 0:
            print(num, " NÃO é um número primo")
            break
    else:
        print(num, " é um número primo.")

else:
    print("Digite um valor maior que 1")

File: exercicio_8.js
let num = 12;

numPrimo(num);

function numPrimo(num) {
  for (let i = 2; i < num; i++) {
    if (num % 2 === 0) {
      return console.log(num + " NÃO é um número primo");
    } else {
      return console.log(num + " é um número primo");
    }
  }
}
